CMakeLists.txt

```diff
@@ -86,7 +86,6 @@ option(BUILD_MEMSTAT "Build rocSPARSE with memory statistics enabled" OFF)
 option(BUILD_ROCSPARSE_ILP64 "Build rocSPARSE with rocsparse_int equal to int64_t" OFF)
 option(BUILD_COMPRESSED_DBG "Enable compressed debug symbols" ON)
 option(BUILD_WITH_ROCBLAS "Enable building rocSPARSE with rocBLAS" ON)
-option(BUILD_WITH_ROCTX "Enable building rocSPARSE with rocTX" ON)
```

Comment:
This is generally something that is nice to be able to control explicitly. My favourite behaviour is `-DBUILD_WITH_ROCTX=ON` means error if roctx not found, `-DBUILD_WITH_ROCTX=OFF` means don't use roctx, and unspecified means search for roctx and use it if found (but don't error if it's not found).

With that said, the way you've done it will work fine for me. I don't actually _need_ finer-grained control.